com.unity.render-pipelines.core/Runtime/Common/CoreAttributes.cs

```diff
@@ -2,6 +2,36 @@
 
 namespace UnityEngine.Rendering
 {
+    /// <summary>
+    /// This attribute allows you to add information for a class to be supported on a render pipeline
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class SupportedOnAttribute : Attribute
+    {
+        /// <summary>
+        /// The pipeline types that define the support
+        /// </summary>
+        public Type[] pipelineTypes { get; }
+
+        /// <summary>
+        /// Creates a new <seealso cref="SupportedOn"/> instance.
+        /// </summary>
+        /// <param name="pipelineTypes">The list of pipeline types
+        /// create sub-menus.</param>
```

Comment:
"create sub-menus" - is this a copypaste bug?

Reply:
yes, thanks,